terraform/20-app/cognito.tf

```diff
@@ -0,0 +1,120 @@
+resource "aws_iam_role" "cognito_sns_role" {
+  name = "${local.prefix}-cognito-sns-role"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17",
+    Statement = [
+      {
+        Effect = "Allow",
+        Principal = {
+          Service = "cognito-idp.amazonaws.com"
+        },
+        Action = "sts:AssumeRole"
+      }
+    ]
+  })
+}
+
+resource "aws_sns_topic" "cognito_topic" {
+  name = "${local.prefix}-cognito-sms-topic"
+}
+
+resource "aws_iam_policy" "cognito_sns_policy" {
```

Comment:
Could we use the `terraform-aws-modules/sns/aws` terraform module for the SNS topic here? And bundle in the iam policies in there as well?

Reply:
Yes now using sns terraform module